project-base/docker/php-fpm/docker-php-consumer-entrypoint

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+default_tries_before_fail=100
```

Comment:
Why `default_tries_before_fail`? Is there any way to change it? Imo it should have been just `trues_before_fail`.

Reply:
renamed in f6a9f4721f8287157f825343b73ec2011353ccd9